compiler/circle-mpqsolver/src/pattern/PatternResolver.cpp

```
@@ -0,0 +1,134 @@
+/*
+ * Copyright (c) 2023 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "PatternResolver.h"
+
+#include <luci/IR/CircleNodes.h>
+
+using namespace mpqsolver::pattern;
+
+bool Q8LayerNormWithQ16VarianceResolver::resolve(luci::Module *module,
+                                                 std::map<std::string, LayerParam> &nodes_params)
+{
+  if (!module)
+  {
+    return false;
+  }
```

It would be better to throw an exception.